pkg/config/node.go

```diff
@@ -313,17 +321,30 @@ func GetLBConfig(kubeconfigPath string, apiPort, lbPort, statPort uint16, apiVip
 		LbPort:   lbPort,
 		StatPort: statPort,
 	}
+
 	// LB frontend address: IPv6 '::' , IPv4 ''
 	if apiVip.To4() == nil {
 		config.FrontendAddr = "::"
 	}
-
-	backends, err := getSortedBackends(kubeconfigPath)
+	// since:
+	// 1: Keepalived script for Api vip checks HAProxy LB status in addition to localhost-kube-apiserver.
```

Comment:
I don't know that these comments are still relevant. The loadbalanced healthchecks work fine with the API VIP. Also, the current version of them is smart enough to handle the case where no haproxy instances are running but a local apiserver is.